python/cugraph/__init__.py

```diff
@@ -31,6 +31,9 @@
 
 from cugraph.snmg.link_analysis.mg_pagerank import mg_pagerank
 
+from cugraph.proto.component import strong_connected_component
```

Comment:
should this be components instead of component

Reply:
yep :(